src/plugins/showHiddenChannels/index.tsx

```diff
@@ -492,6 +492,16 @@ export default definePlugin({
         }
     ],
 
+
+    fixPermCheck(originalPerms: any, channel: Channel) {
```

Comment:
you're doing math on this so it's either a number or this code is wrong

```suggestion
    fixPermCheck(originalPerms: number, channel: Channel) {
```